docs/tutorial/kymotracking.rst

```diff
@@ -94,14 +94,14 @@ Centroid
 
 Once we are happy with the tracks found by the algorithm, we may still want to refine them. Since the algorithm finds
 tracks by determining local peaks and stringing these together, it is possible that some scan lines in the kymograph
-don't have an explicit point on the track associated with them. Using :func:`~lumicks.pylake.refine_lines_centroid` we
+don't have an explicit point on the track associated with them. Using :func:`~lumicks.pylake.refine_tracks_centroid` we
 can refine the tracks found by the algorithm. This function interpolates the tracks such that each time point gets its
 own point on the track. Subsequently, these points are then refined using a brightness weighted centroid. Let's perform
-line refinement and plot the longest track::
+track refinement and plot the longest track::
 
     longest_track_idx = np.argmax([len(track) for track in tracks])  # Get the longest track
 
-    refined = lk.refine_lines_centroid(tracks, line_width=0.3)
+    refined = lk.refine_tracks_centroid(tracks, track_width=0.3)
```

Comment:
Shouldn't the width now be different?

Reply:
It was wrong before because recently I changed it not realizing that `refine_lines_centroid()` expected pixel value. I think having everything working in consistent units now is a lot less confusing